bin/seeds.js

```diff
@@ -0,0 +1,107 @@
+// To insert in "bin/seeds.js"
+const mongoose = require('mongoose');
+const Movie = require('../models/Movie.model');
+
+const DB_NAME = 'express-cinema-dev';
+
+mongoose.connect(`mongodb://localhost/${DB_NAME}`, {
```

Comment:
The connection string to your database should be in the .env file (place this file at the root of your project)